Library/ShoKanri.Http/Responses/Account/GetAccountByIdResponse.cs

```diff
@@ -1,2 +1,2 @@
 namespace ShoKanri.Http.Responses.Account;
-public record GetByIdAccountResponse(string Name,int UserId, decimal Balance, string Description);
+public record GetAccountByIdResponse(string Name,int UserId, decimal Balance, string Description);
```

Comment:
```suggestion
public record GetAccountByIdResponse(string Name, decimal Balance, string Description);
```
O parâmetro `UserId` será fornecido pela rota.